user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md

```diff
@@ -0,0 +1,150 @@
+---
+uid: Register_Catalog_Item
+---
+
+# Registering a Catalog item
+
+## About Catalog items
```

Comment:
Could we move line 7 to 32 to a separate page, below "About the Catalog module"? That way we separate the concept information from the actual how-to, which will probably be more user-friendly, especially since the concept info could also be relevant for other topics.

Reply:
Do we need to add this to the "About the Catalog module" page instead?
Do not really know what title we would give the page otherwise.

Reply:
Yes, that's also a valid option. Then we can create a subsection "Accessing the Catalog module" with the info that's already there, and "About Catalog items" with this info.